backend/tests/helpers/test_app.py

```diff
@@ -101,6 +101,7 @@ async def register_core_schema(
         schema = SchemaRoot(**core_models)
         schema_branch = registry.schema.register_schema(schema=schema, branch=default_branch.name)
         default_branch.update_schema_hash()
+        await registry.schema.update_schema_branch(schema=schema_branch, db=db, branch=default_branch.name)
```

Comment:
Needed because without it, if we load some schemas with unidirectional relationships towards core schemas (like `BuiltinTag`), then these core schemas would be updated in the db because their field `unidirectional_relationships` will change, but as they were not loaded previously their `id` is not defined and thus it will fail.